python/runtime/cudaq/algorithms/py_run.cpp

```diff
@@ -60,16 +60,16 @@ getKernelLaunchParameters(py::object &kernel, py::args args) {
 
   // Lookup the runnable kernel.
   auto mod = unwrap(kernelMod);
-  auto runKern = mod.lookupSymbol<func::FuncOp>(
+  auto runKern = mod.lookupSymbol<mlir::func::FuncOp>(
```

Comment:
For clarity: With an implementation file (`.cpp`), it is acceptable to have a single `using namespace` _foo_`;` at the top of the module after the `#include` directives.